### PR TITLE
Filling in layout home page, login page and signup page

### PR DESCRIPTION
**Pull request by lumosnysm**

### CODE REVIEW

**Comment by lumosnysm** [comment]
ready

**Comment by lumosnysm** [comment]
ready

**Comment by lumosnysm** [comment]
ready

**Comment by lumosnysm** [comment]
ready

File: config/locales/vi.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+vi:
```
Comment (huongnguyenmta) [inline config/locales/vi.yml]: viết luôn đi em ơi =))

**Comment by huongnguyenmta** [comment]
commented

**Comment by lumosnysm** [comment]
ready

**Pull request merged**